build-system/test-configs/forbidden-terms.js

```diff
@@ -605,21 +605,20 @@ const forbiddenTermsGlobal = {
       'src/service/resources-impl.js',
     ],
   },
-  '__AMP_EXP': {
+  '\\b__AMP_EXP\\b': {
     message:
       'Do not access __AMP_EXP directly. Use isExperimentOn() to access config',
     allowlist: [
       'src/experiments/index.js',
       'src/experiments/experiments.extern.js',
     ],
   },
-  'AMP_EXP': {
+  '\\bAMP_EXP\\b': {
```

Comment:
nit for a follow-up: just one rule `'\\b(__)?AMP_EXP\\b'`

Reply:
Ugh this is _almost_ done passing the checks... wanna put that in your followup PR? :D